config/runtime/tests/preempt-rt.jinja2

```diff
@@ -14,13 +14,13 @@
           steps:
           - apt-get update && apt-get install -y procps rt-tests
       name: preempt-rt-prereq
-      path: inline/preempt-rt-prereq.yaml
+      path: inline/{{ node.name }}.yaml
 
     - repository: https://github.com/kernelci/test-definitions.git
       from: git
       revision: kernelci.org
-      path: automated/linux/{{ tst_group|default(tst_cmd) }}/{{ tst_cmd }}.yaml
-      name: {{ plan }}
+      path: automated/linux/{{ tst_group|default(tst_cmd) }}/{{ node.name }}.yaml
```

Comment:
Please correct me if I'm wrong - shouldn't this stay `tst_cmd` (from what I can see in https://github.com/kernelci/test-definitions/tree/master/automated/linux and https://github.com/kernelci/kernelci-pipeline/pull/648)?

Reply:
Excellent catch. Fixing.